lib/components/app/default-main-panel.js

```diff
@@ -26,6 +26,7 @@ class DefaultMainPanel extends Component {
         <div style={{
           bottom: showPlanTripButton ? 55 : 0,
           left: 0,
+          overflowY: 'auto',
```

Comment:
There is loss of scrolling ability on the itinerary results without this, so you only see the beginning of long itineraries. Not sure where the loss comes from.